docs/development-environment.md

```diff
@@ -5,8 +5,18 @@ The javascript and CSS components of this plugin's admin interface need to be bu
 **Recommended Environment**
 
 * node 8.x, aka `lts/carbon`
-* npm 3.8.x
-* yarn 0.24.4
+* npm latest that comes with node (eg 5.5.1 comes with 8.9.3)
```

Comment:
We don't actually use `npm` so Maybe it doesn't have a place here as a recommended tool